backend/src/main/java/gov/cdc/usds/simplereport/service/ScheduledTasksService.java

```diff
@@ -0,0 +1,36 @@
+package gov.cdc.usds.simplereport.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+
+@Service
+public class ScheduledTasksService {
+    private static final Logger LOG = LoggerFactory.getLogger(ScheduledTasksService.class);
+    private final DataHubUploaderService _dataHubUploaderService;
+
+    ScheduledTasksService(DataHubUploaderService dataHubUploaderService) {
+        _dataHubUploaderService = dataHubUploaderService;
+    }
+
+    // Example of how to run periodically, save for future use (and testing)
+    private static final int FIXED_DELAY_MS = 1000*60*30;  // fixed period in milliseconds after last task ends
```

Comment:
Could we use the `Duration` class to make this a bit clearer? Or, maybe given the time in the comment, so we don't have to convert in our heads (which I always screw up 😉 ).